docs/versions/index.md

```diff
@@ -0,0 +1,124 @@
+
+This approach improves our documentation in several ways: 
+
+* When a user arrives in our documentation from an outside source, they'll be likelier to land on a page or section that applies to them.
+* There is a single "source of truth" for each feature, which helps us to maintain consistency, accuracy, and maintainability of our documentation over time, and avoids "drift" between multiple similar sets of documentation.
+* Comparing and contrasting differences helps users to understand what is available to them, and improves awareness of the ways certain offerings might improve their experience.
+
+::::{note}
+This page documents the first version of our approach to documenting differences in versions and deployment types, using our current technologies. 
+Our approach might change as additional documentation features become available.
+::::
+
+## Basic concepts and principles
+
+* [Versioning facets](#versioning-facets)
+* [Defaults and hierarchy](#defaults-and-hierarchy)
+* [Versions and lifecycle states](#versions-and-lifecycle-states)
+
+### Versioning facets
+There are multiple facets to versioning that we need to consider: 
+
+* **Elasticsearch / Kibana flavor:** The feature base used for basic functionality. Either **Serverless** (sometimes "Elastic Cloud Serverless") or **Stack <version>**
+
+% TODO: Final term for "Stack"
+* **Deployment type:** The way Elastic is deployed: 
+  * **Serverless** (sometimes "Elastic Cloud Serverless")
+  * **Elastic Cloud Hosted**
+  * **Elastic Cloud on Kubernetes**
+  * **Elastic Cloud Enterprise**
+  * **Self-managed**
+
+  All deployment types other than **Serverless** use the **Stack <version>** flavor of Elasticsearch / Kibana.
+
+% TODO: Final term for "Self-managed"
+
+* **Project type:** The Serverless project types where a feature can be used - either **Elasticsearch**, **Security**, or **Observability**.
+
+* **Other versioning schemes:** Elastic products or tools with a versioned component, where stack versioning is not followed. 
+  
+  E.g. clients, Elastic Common Schema
+
+**How many facets do I need to use?**
+
+The role of these labels is providing a trust signal that the reader is viewing content that’s applicable to them. This means that the relevant labels should appear on all pages. However, we can choose to expose only one versioning plane on pages where only one plane is relevant:
+
+* Depending on what you're documenting, you might need to include information from multiple facets. For example, when relevant features exist at both the stack and the deployment level, both of these groups might be used together (e.g. security, user management, trust).
+
+* In some sections, such as **Explore and analyze**, features generally only differ by Elasticsearch flavor. In these cases, you can choose to include only this facet on the page.
```

Comment:
> * In some sections, such as **Explore and analyze**, features generally only differ by Elasticsearch flavor. In these cases, you can choose to include only this facet on the page.

This implies that any page that doesn't specify one of the facets applies to all the rest. If we're going to go this very clear.

Reply:
not 100% sure how to address this